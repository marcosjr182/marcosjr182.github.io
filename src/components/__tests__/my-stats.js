import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Stats from '../my-stats';

const survivor = {
  name: 'TestName',
  items: {
    Water: 2,
    Food: 1,
    Ammunition: 4,
    Medication: 3
  }
}

const wrapper = shallow(<Stats user={survivor} />);

describe('Report component', () => {
  it('should render info correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})
