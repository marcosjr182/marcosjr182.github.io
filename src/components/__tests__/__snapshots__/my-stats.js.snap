exports[`Report component should render info correctly 1`] = `
<div
  className="col-xs-12 col-sm-8">
  <Stats
    user={
      Object {
        "items": Object {
          "Ammunition": 4,
          "Food": 1,
          "Medication": 3,
          "Water": 2,
        },
        "name": "TestName",
      }
    } />
</div>
`;
